### ISSUE TITLE
Transfers between affiliates' virtual accounts

### ISSUE THREAD

**Issue by TiberiusB** [valnet/valuenetwork#503]
I think this sort of transaction is pretty interesting, and I don't know if it is possible now... 

Affiliates can settle dept between them. They can borrow money, let's say if someone has from a project, and get cash from CAKE.

**Comment by fosterlynn** [comment]
Yes, an interesting use case.  Note Enspiral does things this way too. :)

You can create an exchange type that is configured for this type of transfer.
